### ISSUE TITLE
Anti-thyroglobulin antibody positivity

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#7354]
**Preferred term label:**
Anti-thyroglobulin antibody positivity

**Synonyms**
TgAbs, Anti-thyroid globulin antibody positivity, TGA

**Definition (free text, please give PubMed ID)**
PMID: 33144894, PMID: 34634575, PMID: 33999913

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Hashimoto's thyroiditis

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by MaudDLVT** [comment]
@julienN75

**Comment by pnrobinson** [comment]
This term exists as [Anti-thyroglobulin antibody positivity HP:0032069](https://hpo.jax.org/app/browse/term/HP:0032069).
I have added the synonyms and the citations.
